engine-rest/engine-rest-openapi/src/main/templates/paths/history/batch/get.ftl

```diff
@@ -0,0 +1,76 @@
+<#-- Generated From File: camunda-docs-manual/public/reference/rest/history/batch/get-query/index.html -->
+<#macro endpoint_macro docsUrl="">
+{
+  <@lib.endpointInfo
+      id = "getHistoricBatches"
+      tag = "Historic Batch"
+      summary = "Get Historic Batches"
+      desc = "Queries for historic batches that fulfill given parameters. Parameters may be
+              the properties of batches, such as the id or type. The
+              size of the result set can be retrieved by using the
+              [Get Historic Batch Count](${docsUrl}/reference/rest/history/batch/get-query-count/)
+              method."
+  />
+
+  "parameters" : [
+
+    <#assign last = false >
+    <#assign requestMethod="GET"/>
```

Comment:
I think we don't need the request method at the moment since we don't use this in the query-params file.